_includes/head.html

```diff
@@ -13,23 +18,70 @@
-      g.shutdown=function(){g("rec",!1)};g.restart=function(){g("rec",!0)};
-      g.consent=function(a){g("consent",!arguments.length||a)};
-      g.identifyAccount=function(i,v){o='account';v=v||{};v.acctId=i;g(o,v)};
-      g.clearUserCookie=function(){};
-    })(window,document,window['_fs_namespace'],'script','user');
+    (function (m, n, e, t, l, o, g, y) {
+      if (e in m) {
+        if (m.console && m.console.log) {
+          m.console.log(
+            'FullStory namespace conflict. Please set window["_fs_namespace"].'
+          );
+        }
+        return;
+      }
+      g = m[e] = function (a, b, s) {
+        g.q ? g.q.push([a, b, s]) : g._api(a, b, s);
+      };
+      g.q = [];
+      o = n.createElement(t);
+      o.async = 1;
+      o.src = 'https://' + _fs_host + '/s/fs.js';
+      y = n.getElementsByTagName(t)[0];
+      y.parentNode.insertBefore(o, y);
+      g.identify = function (i, v, s) {
+        g(l, { uid: i }, s);
+        if (v) g(l, v, s);
+      };
+      g.setUserVars = function (v, s) {
+        g(l, v, s);
+      };
+      g.event = function (i, v, s) {
+        g('event', { n: i, p: v }, s);
+      };
+      g.shutdown = function () {
+        g('rec', !1);
+      };
+      g.restart = function () {
+        g('rec', !0);
+      };
+      g.consent = function (a) {
+        g('consent', !arguments.length || a);
+      };
+      g.identifyAccount = function (i, v) {
+        o = 'account';
+        v = v || {};
+        v.acctId = i;
+        g(o, v);
+      };
+      g.clearUserCookie = function () {};
+    })(window, document, window['_fs_namespace'], 'script', 'user');
   </script>
-  <meta charset="utf-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>{% if page.title %}{{ page.title }} | {% endif %} {{ site.title }}</title>
-  <meta name="description" content="{{ site.description }}">
-  <link rel="icon" href="https://cdn.zapier.com/zapier/images/favicon.ico">
-  <link rel="stylesheet" href="{{ site.baseurl }}/assets/stylesheets/base.css">
+  <script
+    type="module"
+    src="https://cdn.zapier.com/packages/partner-sdk/v0/zapier-elements/zapier-elements.esm.js"
+  ></script>
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <title>
+    {% if page.title %}{{ page.title }} | {% endif %} {{ site.title }}
+  </title>
+  <meta name="description" content="{{ site.description }}" />
+  <link rel="icon" href="https://cdn.zapier.com/zapier/images/favicon.ico" />
+  <link
+    rel="stylesheet"
+    href="{{ site.baseurl }}/assets/stylesheets/base.css"
+  />
+  <link
+    rel="stylesheet"
+    href="https://cdn.zapier.com/packages/partner-sdk/v0/zapier-elements/zapier-elements.css"
+  />
```

Comment:
This file got formatted on save, so the actual additions are `zapier-elements.css` and `zapier-elements.esm.js`

Reply:
Guessing your editor (VS most likely?) has some global formatter thing setup since I don't see one configured in this repo. Thinking out loud here since two things crossed my mind:

1. Formatter is always welcome. Should it be configured for this repo specifically if we haven't already rather than using local settings?
2. I guess the scripting tags formatted are a little unusual. They are copied from vendors and compact in the first place. Do you think of any advantages for one over the other?

Not a blocker though.

Reply:
Yeah good points:

1. Thinking through this, this repo is mostly markdown files and I think folks tend to edit them in GitHub rather than an IDE, so adding a formatter might be a moot point..
2. AFAIK the compact code just reduces whitespace and therefore makes the script tag take up less space. Even when formatted it's not easy to read so using less memory is the only advantage I'd say

Thanks for approving this btw!

Reply:
Sure thing @Jesswinters !

I think one good thing about formatters is consistency between edits. Without a formatter there are no formatting rules essentially. Local settings of contributors compete with each other and increase the noise ratio. Yup, some folks update the docs in GH directly. However, I'd say it's not the recommended path since we don't have a PR preview set up.

I brought this up since the changes due to formatting made me wonder how they were introduced in the first place. It feels a little odd seeing them committed since we don't have a formatter in place yet. Even if we had a formatter in place, I think we should skip formatting them still.

> Even when formatted it's not easy to read

It's not meant to be read, by humans 😄. Most scripts from vendors are mangled and minified.

Reply:
Agreed on all of this haha. I almost added a formatter file to this repo but hopefully these docs can be moved to a CMS so it's easier to update in the future! Maybe next time I make a change here I'll just setup a formatter file haha